raygun4go_test.go

```diff
@@ -35,6 +35,36 @@ func TestClient(t *testing.T) {
 			So(err, ShouldEqual, nil)
 		})
 
+		Convey("#Clone", func() {
+			t := []string{"Critical", "Urgent", "Fix it now!"}
+			c.Tags(t)
+
+			cd := "foo"
+			c.CustomData(cd)
+
+			r := &http.Request{}
+			c.Request(r)
+
+			v := "1.2.3"
+			c.Version(v)
+
+			u := "user"
+			c.User(u)
+
+			clone := c.Clone()
```

Comment:
Probably want to make a change *after* the clone and ensure it doesn't propagate?

Reply:
Good idea